### DISCUSSION TITLE
Funnel type chart

### DISCUSSION THREAD

**Discussion by geordidearns** [tremorlabs/tremor-npm#721] [Ideas]
Hey team, love the updates and watching this project grow!

Likely been suggested, but a funnel type of chart would be very helpful to visualise the flow of data, perhaps represented by passing in an array representing a step/stage, and rendering a horizontal or vertical funnel?

thanks for considering it either way!

**Comment by Guillecasas1** [comment]
Hi all, the idea to have tremor funnels sounds awesome! Big up 🙌🏻
